src/main/java/Exceptions/UnknownCommandException.java

```diff
@@ -0,0 +1,10 @@
+package exceptions;
+
+/**
+ * Raised exception on unknown command input.
```

Comment:
Just a suggestion 🤔 Would it be better to change the javadoc description from "Raised" to "Raises" as according to the [javadoc guideline in Week 3 topics](https://nus-cs2103-ay2425s1.github.io/website/schedule/week3/topics.html)